src/InputWithLabel.js

```diff
@@ -0,0 +1,26 @@
+
+import React from "react";
+   const InputWithLabel = ({
+      value,
+      handleTitleChange,
+      isFocused,
```

Comment:
Great job creating the inpuRef variable and storing the useRef() hook!

I see that the input element is not focus after adding an item or refreshing the page. I leave some suggestions below.

- I believe the isFocused prop is not needed here since it isn't passed down from the AddTodoForm component - if you want to try and console the prop it will be undefined, I would get rid of it.